test-loop-tests/src/tests/catching_up.rs

```diff
@@ -375,27 +371,11 @@ fn ultra_slow_test_catchup_sanity_blocks_produced() {
 }
 
 #[test]
-fn ultra_slow_test_all_chunks_accepted_1000() {
-    test_all_chunks_accepted_common(1000, 3000, 7)
-}
-
-#[test]
-fn ultra_slow_test_all_chunks_accepted_1000_slow() {
-    test_all_chunks_accepted_common(1000, 6000, 7)
-}
-
-#[test]
-fn ultra_slow_test_all_chunks_accepted_1000_rare_epoch_changing() {
```

Comment:
We don't need the other configurations. Also, we don't need 1000 blocks, ~100 should be sufficient for this test.

Reply:
Why do we not need other configurations?

Reply:
The difference in configuration 1 and 2 was just the block production time of 3 sec vs 6 sec, which are both irrelevant now. The third configuration was testing with a longer epoch length, which again I feel isn't necessary as smaller epoch lengths and hence more number of epochs lead to more validator rotations which is what we'd like to check.